environment/k8s_environment.go

```diff
@@ -51,7 +51,7 @@ type K8sEnvSpecs []K8sEnvResource
 
 // K8sEnvSpecInit is the initiator that will return the name of the environment and the specifications to be deployed.
 // The name of the environment returned determines the namespace.
```

Comment:
Does this comment line still make sense?

Reply:
No, we switched our namespace naming to the inputs in the suite setup a bit back, e.g.
```go
suiteSetup, err = actions.MultiNetworkSetup(
  "multi-network-chainlink",       // <<<<------------- Namespace initialization
  environment.NewChainlinkCluster(0),
  client.MultipleNetworks("ethereum_geth", "ethereum_hardhat"),
  tools.ProjectRoot,
)
```
The string that was returned by the `K8sEnvSpecInit` functions was never used, so I removed it on autopilot, assuming it was vestigial. But now I'm thinking about it more, it's a bit odd needing to supply the namespace at this level, so I'm going to look at why we delineated from the old ways.

Reply:
Ah, when we split environment creation and spec deployment into separate parts, it necessitated this new style. I'd still like to condense it though.